Introduction to Mastering Github campus

azrad opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

nurulazradb committed:
Introduction to Mastering Github campus

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+Hi my name is Nurulazrad and I'm a Code School student.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

nurulazradb committed:
Love Ruby Path

nurulazradb committed:
Deadly Skills

nurulazradb committed:
Add killing history as skill

codeschool-kiddo committed:
Added new rule about coffee

nurulazradb committed:
Fix coffee and downcase Dojo

nurulazradb committed:
Merge pull request #2 from azrad/new_rules

Too many people are spilling coffee

nurulazradb committed:
Merge branch 'master' of https://github.com/azrad/dojo_rules

* 'master' of https://github.com/azrad/dojo_rules:
  Fix coffee and downcase Dojo
  Added new rule about coffee

nurulazradb committed:
Replace assassin names with bad coding practices.

nurulazradb committed:
Update kill list with bad coding practices
v1.0.1 - After some bad coding practices.

* tag 'v1.0.1':
  Replace assassin names with bad coding practices.
  We dont need to kill Budd
  Initial kill list

nurulazradb committed:
Fixes #3 using tabs instead of spaces

nurulazradb committed:
Merge pull request #4 from azrad/kill_list

Fixes #3 using tabs instead of spaces

nurulazradb committed:
All members should read the rules.

nurulazradb committed:
All members should read the rules.

nurulazradb committed:
Merge branch 'master' of https://github.com/azrad/dojo_rules

* 'master' of https://github.com/azrad/dojo_rules:
  All members should read the rules.

Conflicts:
	README.md
